include/mbgl/shaders/mtl/symbol.hpp

```diff
@@ -493,6 +523,22 @@ FragmentStage vertex vertexMain(thread const VertexStage vertx [[stage_in]],
 
     device const SymbolDrawableUBO& drawable = drawableVector[uboIndex];
 
+    const float2 fade_opacity = unpack_opacity(vertx.fade_opacity);
+    const float fade_change = (fade_opacity[1] > 0.5) ? paintParams.symbol_fade_change : -paintParams.symbol_fade_change;
+    const half fo = half(max(0.0, min(1.0, fade_opacity[0] + fade_change)));
+
+)"
+#if MTL_SDF_SHADER_VERTEX_CULLING_ENABLED
+                                   R"(
+    if (fo == 0.0) {
+        return {
+            .position     = float4(-2.0, -2.0, -2.0, 1.0),
```

Comment:
Let's name this position `(-2.0, -2.0, -2.0, 1.0)` in a constant for an easier understanding in the future.